src/submission/validation-clinical/surgery.ts

```diff
@@ -199,3 +188,40 @@ function checkSurgeryTypeEquality(
 
 	return existingSurgeryType === therapyRecordSurgeryType;
 }
+
+// Prevents submitting duplicate records; allows updating existing records with new values
+function checkSurgeryDuplicateOrUpdate(
+	therapyRecord: DeepReadonly<SubmittedClinicalRecord>,
+	prevSurgeries: DeepReadonly<Therapy>[],
+	existingSurgery: DeepReadonly<Therapy> | undefined,
+	errors: SubmissionValidationError[],
+) {
+	// Clone Submitted Record, minus index key, which is not stored on Clinical Records
+	const submissionClone = (({ index, ...rest }) => ({ ...rest }))(therapyRecord);
```

Comment:
This line is clever enough to need its own comment 😛 . I think it works as expected but is there a simpler way, or can we use an `omit` function? This repo already has lodash so just using [`_.omit`](https://lodash.com/docs/2.4.2#omit) would be more readable.

Reply:
Even simpler approach:

```ts
const { index, ...submissionClone } = therapyRecord;
```

Reply:
Destructuring index leaves an unused variable, so I'll go with omit, should have used that in the first place
Updated